Source/Editor/Windows/ContentWindow.cs

```diff
@@ -215,6 +217,20 @@ private ContextMenu OnViewDropdownPopupCreate(ComboBox comboBox)
                         filterButton.Checked = _viewDropdown.IsSelected(filterButton.Text);
                 }
             };
+            
+            var sortBy = menu.AddChildMenu("Sort by");
+            sortBy.ContextMenu.AddButton("Alphabetic Order", OnSortByButtonClicked).Tag = SortType.AlphabeticOrder;
+            sortBy.ContextMenu.AddButton("Alphabetic Reverse", OnSortByButtonClicked).Tag = SortType.AlphabeticReverse;
+            sortBy.ContextMenu.VisibleChanged += control =>
+            {
+                if (!control.Visible)
+                    return;
+                foreach (var item in ((ContextMenu)control).Items)
+                {
+                    if (item is ContextMenuButton button)
+                        button.Checked = _buttonNameChecked == button.Text;
```

Comment:
This could use `(SortType)button.Tag == _sortType` instead of comparison based on button text.